monkey/monkey_island/cc/environment/environment_config.py

```diff
@@ -2,23 +2,34 @@
 
 import json
 import os
-from typing import Dict, List
+from typing import Dict, List, Optional
 
 from monkey_island.cc.environment.user_creds import UserCreds
 from monkey_island.cc.resources.auth.auth_user import User
 from monkey_island.cc.resources.auth.user_store import UserStore
+from monkey_island.cc.server_utils.consts import MONKEY_ISLAND_ABS_PATH
 
 
 class EnvironmentConfig:
     def __init__(self, file_path):
         self._server_config_path = os.path.expanduser(file_path)
         self.server_config = None
-        self.deployment = None
+        self.deployment = self._get_deployment_from_file()
```

Comment:
Since this field is no longer in the `server_config.json`, we can remove it from this class. The only thing that uses this is `version_update.py`. I see no reason why `version_update.py` needs to be coupled to the environment singleton. Nor do I see any reason why all of the other things that are coupled to the environment singleton need access to this field.